README.md

```diff
@@ -405,10 +402,10 @@ $ drive push --destination a1/b2/c3 music/Travi$+Future integrals/complex/compil
 ```
 
 
-Note: To ignore checksum verification during a push:
+To enable checksum verification during a push:
```

Comment:
Thank you! Yap, that information is legacy of when checksum verification was still on by default.